Makefile

```diff
@@ -41,21 +44,24 @@ connscrape:
 unittest:
 	make -C collector unittest
 
-image: collector unittest
+image:
 	make -C collector txt-files
 	docker buildx build --load --platform ${PLATFORM} \
+		--build-arg BUILD_TYPE="$(BUILD_TYPE)" \
+		--build-arg CMAKE_BUILD_TYPE="$(CMAKE_BUILD_TYPE)" \
+		--build-arg USE_VALGRIND="$(USE_VALGRIND)" \
```

Comment:
Does USE_VALGRIND even do anything now that the wrapper script has been removed? If it doesn't, it would be nice to make it work in another PR.

Reply:
It disables profiling [here](https://github.com/stackrox/collector/blob/31ca87ba9f4df8a399b6dd61d8f13f6c39fc82f1/collector/CMakeLists.txt#L18-L20). Think that was due to tcmalloc not being compatible with valgrind because they both overwrite the malloc implementation.

Reply:
I don't know how I missed this when I wrote my answer, but `USE_VALGRIND` is also used for unit tests to be run under valgrind, so it still has a use.

Reply:
I looked at the valgrind unit tests. Valgrind found many errors, but the test passed. They all seemed to have to do with RE2, so maybe it is not a huge concern. If valgrind finds errors, the test should fail. Or are we ignoring the RE2 errors?

Reply:
I remember trying to ignore them but can't remember why I gave up on it, there's a way to set up an ignore file for valgrind (https://valgrind.org/docs/manual/manual-core.html/manual-core.html#manual-core.suppress).

I also remember the errors where in a place that would be really hard to actually fix and didn't seem like too big a deal. IIRC, the error was on the initialization of the re2 regexes that Falco does statically, so if it was a serious issue we would see collector crashing at start up. I'm also pretty sure we don't use the regexes.